crates/consensus/auto-seal/tests/it/main.rs

```diff
@@ -1,4 +1,4 @@
-//! auto-mine consensus tests
-mod auto_mine;
+// //! auto-mine consensus tests
+// mod auto_mine;
```

Comment:
being added bck in this PR?

Reply:
need to update, but likely removing for this PR